Sources/Fuzzilli/Minimization/CallArgumentReducer.swift

```diff
@@ -41,6 +41,15 @@ struct CallArgumentReducer: Reducer {
                 let newOp = CallMethod(methodName: op.methodName, numArguments: op.numArguments - 1)
                 let newInstr = Instruction(newOp, output: instr.output, inputs: Array(instr.inputs.dropLast()))
                 verifier.tryReplacing(instructionAt: instr.index, with: newInstr, in: &code)
+
+            case let op as CallComputedMethod:
+                guard op.numArguments > minArgCount else {
+                    break
+                }
+
+                let newOp = CallComputedMethod(numArguments: op.numArguments - 2)
+                let newInstr = Instruction(newOp, output: instr.output, inputs: Array(instr.inputs.dropLast(2)))
```

Comment:
Why are you dropping two arguments at the same time and not just one like the above cases?

Reply:
That was a mistake on my part. I've corrected it now